src/tests/test_package_loading.py

```diff
@@ -1,86 +1,8 @@
 import pytest
-import shutil
 from pathlib import Path
+import shutil
 
-
-@pytest.mark.parametrize("active_server", ["main", "secondary"])
-def test_load_from_url(selenium_standalone, web_server_secondary, active_server):
```

Comment:
This should still work after the file format change to avoid breaking backward compatibility, unless there is a reason to remove it?

Reply:
Okay fixed.